bin/doctrine.bat

```diff
@@ -1,9 +1,9 @@
-@echo off
-
-if "%PHPBIN%" == "" set PHPBIN=@php_bin@
-if not exist "%PHPBIN%" if "%PHP_PEAR_PHP_BIN%" neq "" goto USE_PEAR_PATH
-GOTO RUN
-:USE_PEAR_PATH
-set PHPBIN=%PHP_PEAR_PHP_BIN%
-:RUN
-"%PHPBIN%" "@bin_dir@\doctrine" %*
+@echo off
+
+if "%PHPBIN%" == "" set PHPBIN=@php_bin@
+if not exist "%PHPBIN%" if "%PHP_PEAR_PHP_BIN%" neq "" goto USE_PEAR_PATH
+GOTO RUN
+:USE_PEAR_PATH
+set PHPBIN=%PHP_PEAR_PHP_BIN%
+:RUN
+"%PHPBIN%" "@bin_dir@\doctrine" %*

```

Comment:
This one is targeted at Windows, and should have Windows line endings I think

Reply:
yes

Reply:
Please do not mark as resolved discussions you did not open. Did you forget to force push or something? The line endings are still changed.

Reply:
> Please do not mark as resolved discussions you did not open.

ok

> Did you forget to force push or something? The line endings are still changed.

yes, because I changed them to Windows/CRLF line endings, thus resolved, right? :)

Reply:
Oh they weren't already? Okay then.